src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -328,11 +329,20 @@ export async function postTransformationJob(userInputState: UserInputState) {
     const durationInMs = calculateTotalLatency(codeTransformTelemetryState.getStartTime())
     const resultStatusMessage = codeTransformTelemetryState.getResultStatus()
 
+    const versionInfoWrapper = await getVersionData('mvnw', transformByQState.getProjectPath())
+    let mavenVersionInfoMessage = versionInfoWrapper[0]
+    let javaVersionInfoMessage = versionInfoWrapper[1]
+    const versionInfo = await getVersionData('mvn', transformByQState.getProjectPath())
+    mavenVersionInfoMessage += ` -- ${versionInfo[0]}`
+    javaVersionInfoMessage += ` -- ${versionInfo[1]}`
```

Comment:
Should we just have one java version?

Reply:
Discussed offline, will keep both for now